### PR TITLE
Add Security Headers

### PR DESCRIPTION
**Pull request by StupidCodeFactory**
**CHANGES:**
Add various headers and cookies to secure HTTP connections
**NOTES:**
Modernizr need to be updated for the PR to be complete

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+GA_TRACKING_ID=UA-96772907-1
```
Comment (alan) [inline .env]: This file should be ignored, how come is commited? I can see it on my local .gitignore...

File: .envrc [hunk 2]
```diff
@@ -0,0 +1 @@
+dotenv
```
Comment (alan) [inline .envrc]: Same for this one

File: config/application.rb [hunk 3]
```diff
@@ -61,5 +61,30 @@ class Application < Rails::Application
...
+      'Content-Security-Policy'   => "default-src 'self'; " \
+                                     'font-src data:; '    \
+                                     "img-src 'self'; "    \
+                                     "object-src 'none'; " \
+                                     "script-src www.google-analytics.com 'self' 'unsafe-eval' " \
+                                     "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=' " + # govuk
+                                     "'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=' " + # govuk
+                                     "'sha256-9GTWoKmlaDM3V+GStWlXFaD4tf+wPfBN2ds2eySQ9aE=' " + # govuk
+                                     "'sha256-jS/ACiWphBsq3cSoeC2ouPyhfJqFFlLNOlwaOVN3UGc=' " + # javascript_i18n
+                                     "'sha256-4KTF3oamE52HQgyVL/chpyuwqC52Jk0kMg+KJiE+YlU=' " + # ga_page_view + @step_name = 'prisoner_step'
```
Comment (alan) [inline config/application.rb]: If these hashes change is there a way that we'll be notified?
Have you looked if this step could be automated as part of of the build or middleware?

File: spec/features/request_a_visit_spec.rb [hunk 4]
```diff
@@ -41,7 +41,6 @@
     click_button 'Continue'
 
     enter_visitor_information email_address: visitor_email
-    select '1', from: 'How many other visitors?'
```
Comment (alan) [inline spec/features/request_a_visit_spec.rb]: What did this do???

**Comment by StupidCodeFactory** [comment]
@alan sorry I wasn't quite done with this PR

File: .envrc (hunk 2 above)
Comment (StupidCodeFactory) [inline .envrc]: .envrc definitely should be checked in. I do not see what is offensive about it.

**Pull request merged**